content/en/docs/concepts/security/index.md

```diff
@@ -632,331 +629,278 @@ authorization result, `ALLOW` or `DENY`.
 
 ### Enabling authorization
 
-You enable Istio Authorization using a `ClusterRbacConfig` object. The `ClusterRbacConfig`
-object is a cluster-scoped singleton with a fixed name value of `default`. You can
-only use one `ClusterRbacConfig` instance in the mesh. Like other Istio configuration
-objects, `ClusterRbacConfig` is defined as a
-Kubernetes `CustomResourceDefinition`
-[(CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) object.
+There is no need to explicitly enable Istio authorization feature, you just apply
+the `AuthorizationPolicy` on **workloads** to enforce the access control.
 
-In the `ClusterRbacConfig` object, the operator can specify a `mode` value, which can
-be:
+If no `AuthorizationPolicy` applies to a workload, no access control will be enforced,
+in other words, all requests will be allowed.
 
-- **`OFF`**: Istio authorization is disabled.
-- **`ON`**: Istio authorization is enabled for all services in the mesh.
-- **`ON_WITH_INCLUSION`**: Istio authorization is enabled only for services and
-  namespaces specified in the `inclusion` field.
-- **`ON_WITH_EXCLUSION`**: Istio authorization is enabled for all services in
-  the mesh except the services and namespaces specified in the `exclusion`
-  field.
+If any `AuthorizationPolicy` applies to a workload, the request will be allowed
+only if it matches the `AuthorizationPolicy`, in other words, the request will be
+denied if it doesn't match any of the `AuthorizationPolicy`, i.e. deny by default.
```

Comment:
If any AuthorizationPolicy applies to a workload, access to that workload is denied by default, unless explicitly allowed by a rule declared in the policy.